_maps/map_files/generic/CentCom.dmm

```diff
@@ -251,11 +251,11 @@
 /turf/open/space/basic,
 /area/space)
 "fX" = (
-/turf/closed/indestructible/riveted,
+/turf/closed/indestructible/start_area,
 /area/start)
 "fY" = (
 /obj/effect/landmark/start/new_player,
-/turf/open/floor/plating,
+/turf/closed/indestructible/start_area,
```

Comment:
mobs can't breath in closed turfs without an air tank. Are you sure this won't be a problem?

Reply:
i... maybe want to go back on this? just in case

Reply:
decided?